index.js

```diff
@@ -56,15 +58,22 @@ module.exports = function(ServerlessPlugin) {
      */
 
     _optimize(evt) {
-
       let _this = this;
+      let conf = _.merge({}, {
+        browserify: {
+            plugins: [],
+            transforms: [],
+            exclude: [],
+            ignore: [],
+        }
+      }, this.S._projectJson.custom.optimize, evt.function.custom.optimize);
 
       // Skip plugin if this function is not optimized
-      if (!evt.function.custom.optimize) return BbPromise.resolve(evt);
+      if (!conf) return BbPromise.resolve(evt);
```

Comment:
This won't work anymore and has to be moved before the merge and changed like this:

```
if (!this.S._projectJson.custom.optimize && !evt.function.custom.optimize) return BbPromise.resolve(evt);
```